pkg/es/config/config.go

```diff
@@ -541,7 +556,40 @@ func GetHTTPRoundTripper(c *Configuration, logger *zap.Logger) (http.RoundTrippe
 	return transport, nil
 }
 
-func loadToken(path string) (string, error) {
+func watchPasswordFromFile(path string, passwordChan chan<- string) error {
```

Comment:
@haanhvu so this is exactly the path that I did not go through. Please see the new comment on the ticket: https://github.com/jaegertracing/jaeger/issues/3924#issuecomment-1428567250